projects/core/src/occ/adapters/cart/occ-cart-entry.adapter.ts

```diff
@@ -43,7 +42,7 @@ export class OccCartEntryAdapter implements CartEntryAdapter {
     return this.http
       .post<CartModification>(url, toAdd, { headers, params })
       .pipe(
-        catchError((error: any) => throwError(error.json())),
+        catchError((error: any) => throwError(error)),
```

Comment:
Is this line changing anything?
It looks like we are we catching an error only to rethrow it again.

Reply:
Besides removing `.json()`, I didn't want to change anything as I'm not sure what's the idea behind rethrowing this error here.

Reply:
But by "only" removing ` .json()` you are changing everything that this line does ;-) So:
1) you should have the idea what this line does
2) why keep obsolete/redundant code that does nothing?

Reply:
We checked that `.json()` does not exist in the http error response, thus the removal and no need for me to dig deeper than that.